README.md

```diff
@@ -1,64 +1,99 @@
 # MovR
 
-MovR is a fictional ride sharing company.
+MovR is a fictional vehicle-sharing company.
+
+This repo contains the source code for the Python implementation of the MovR load generator.
 
 ## Getting started
-First, [download CockroachDB](https://www.cockroachlabs.com/docs/stable/install-cockroachdb.html) and start a local cluster with `cockroach start --insecure --host localhost --background`
 
-Then create the database `movr` with `cockroach sql --insecure --host localhost -e "create database movr;"`
+First, [download CockroachDB](https://www.cockroachlabs.com/docs/v21.1/install-cockroachdb.html) and [start a local, single-region, 3-node cluster](https://www.cockroachlabs.com/docs/v21.1/cockroach-start.html):
+
+```
+$ cockroach start --insecure --store=node1 --listen-addr=localhost:26257 --http-addr=localhost:8080 --join=localhost:26257,localhost:26258,localhost:26259 --background --locality=region=us-east-1 --cache=.25 --max-sql-memory=.25
+
+$ cockroach start --insecure --store=node2 --listen-addr=localhost:26258 --http-addr=localhost:8081 --join=localhost:26257,localhost:26258,localhost:26259 --background --locality=region=us-east-1 --cache=.25 --max-sql-memory=.25
+
+$ cockroach start --insecure --store=node3 --listen-addr=localhost:26259 --http-addr=localhost:8082 --join=localhost:26257,localhost:26258,localhost:26259 --background --locality=region=us-east-1 --cache=.25 --max-sql-memory=.25
+
+$ cockroach init --insecure
+```
+
+Then, use the [`cockroach` SQL client](https://www.cockroachlabs.com/docs/v21.1/cockroach-sql.html) to create the `movr` database with the `movr_init.sql` file:
```

Comment:
this is supported via the movr load command - we shouldn't need to redirect any SQL into an empty database